_data/links.yml

```diff
@@ -2,5 +2,4 @@ twitter: https://twitter.com/OvercastPGM
 discord: https://oc.tc/discord
 twitch: https://twitch.tv/OCNLive
 youtube: https://www.youtube.com/c/OvercastNetwork
-github: https://github.com/OvercastCommunity
-donate: https://oc.tc/donate
+github: https://github.com/OvercastCommunity
```

Comment:
Fix line endings if possible, otherwise I can do it later. Should be Unix line endings.